### PR TITLE
v2.0.0

### PR DESCRIPTION
**Pull request by stoe**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -40,72 +41,66 @@ async function* getOrganizations(octokit, enterprise = '', cursor = null, record
...
+    const date = dayjs().toISOString()
 
     const opts = {
       owner,
       repo,
-      path,
+      path: reportPath,
       message: `${date} invitation report`,
       content: Buffer.from(csv).toString('base64'),
       committer: {
```
Comment (droidpl) [inline index.js:106]: Should the committer params be customizable? Otherwise maybe they want to link those to a service account

**Review by droidpl — APPROVED** [review]
All looks pretty solid 😄  Just added a minor idea in case you want to add it

File: index.js (hunk 1 above)
Comment (stoe) [reply index.js:106]: Great idea, wanna start a PR, @droidpl?

**Review by froi — CHANGES_REQUESTED** [review]

**Review by froi — APPROVED** [review]
Very nice @stoe :shipit:

**Pull request merged**